### DISCUSSION TITLE
githubWebHookServer support non-https

### DISCUSSION THREAD

**Discussion by knguyenst** [actions/actions-runner-controller#1519] [Questions]
Hello ARC Team,

I have ALB handling https termination to our Kubernetes cluster with istio internallly to port 80.  ARC webhook server is listening on port 9443 https by default.

I cannot find a way for it to listen to regular http port w/o https.

Regards,

K

**Comment by rjeczalik** [comment]
Are you sure it's `github-webhook-server` which listens on 9443 https?

I use ALB termination as well, github-webhook-server container runs using 0.24 and it listens on 8000 http:

```
    Port:          8000/TCP
    Host Port:     0/TCP
    Command:
      /github-webhook-server
    Args:
      --metrics-addr=127.0.0.1:8080
      --sync-period=5m
```

It's the `manager` container from actions-runner-controller that listens on 9443.
